src/microservices/users/tests/FunctionalTests/Base/FunctionalTestBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace FunctionalTests.Api.Base
+{
+    using System;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Logging;
+    using FunctionalTests.Api.Helpers;
+    using Grpc.Net.Client;
+    using global::Api;
+    using DotNet.Testcontainers.Containers.Modules.Databases;
+    using DotNet.Testcontainers.Containers.Builders;
+    using DotNet.Testcontainers.Containers.Configurations.Databases;
+    using Persistence;
+    using Microsoft.EntityFrameworkCore;
+
+    // https://github.com/grpc/grpc-dotnet/blob/master/examples/Tester/Tests/FunctionalTests/FunctionalTestBase.cs
```

Comment:
Maybe remove that comment?

Reply:
I left the comment on purpose so we can reference where the source comes from, but I can remove it if you think it wouldn't be helpful